### PR TITLE
CS-28: Arduino ESP8266 v3

### PR DESCRIPTION
**Pull request by deltalejo**
Use BearSSL API for WiFiClientSecure instead of the deprecated one AxTLS.
Version jumps from 3.0.1 to 3.0.3 because in the repo exists the tag v3.0.2.

### CODE REVIEW

File: src/UbiConstants.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ Inc
 #include "stdint.h"
 
 const char UBIDOTS_INDUSTRIAL_IP[] = "169.55.61.243";
-const char *const USER_AGENT = "UbidotsESP8266/3.0.1";
+const char *const USER_AGENT = "UbidotsESP8266/3.0.3";
```
Comment (woakas) [inline src/UbiConstants.h]: I think that this change is minor so, I suggest that the version be 3.1.0

**Comment by woakas** [comment]
@deltalejo I think an `include` is needed to the BearSSL library, could you confirm that this include is not needed?

**Review by woakas — COMMENTED** [review]
@deltalejo  I think an include is needed to the BearSSL library, could you confirm that this include is not needed?

File: src/UbiConstants.h (hunk 1 above)
Comment (deltalejo) [reply src/UbiConstants.h]: Appropriate header files for BearSSL are indirectly included by [ESP8266WiFi.h](https://github.com/esp8266/Arduino/blob/612e7ffd7fb398fc2cdcf8bd1d177b75d997d46c/libraries/ESP8266WiFi/src/ESP8266WiFi.h#L42) which is included by [src/UbiProtocol.h](https://github.com/ubidots/ubidots-esp8266/blob/a83879a030ee5f801c857c3ea8afcee9e94957d3/src/UbiProtocol.h#L28).
Version 3.1.0 is OK.

**Review by woakas — APPROVED** [review]

**Pull request merged**